groups/rds/variables.tf

```
@@ -45,6 +45,10 @@ variable "rds_databases" {
   type = map
 }
 
+variable "rds_ingress_groups" {
+  type = map
```

Can we tighten up the type constraint here to enforce the expected data structure and add a suitable variable description:

```suggestion
  type        = map(list(string))
  description = "A map whose keys represent RDS instances and whose values are lists of strings representing security group filter patterns"
```